source/_patterns/components/03-cards/01-card-footer-dropdown.hbs

```diff
@@ -1 +1 @@
-{{> components-card cardTitleSize="h4" cardTitle="Card Footer with Dropdown" cardContents="This card has a Footer and a dropdown menu in the footer." cardFooter=true cardFooterContents="Card Footer" cardFooterDropdown=true }}
```

Comment:
I wrapped the footer contents with <p>, which is something I noticed here: http://www.patternfly.org/pattern-library/dashboard/dashboard-card/#/api

I'm not sure if this is the best approach, but I struggled with the spacing in the footer when a dropdown is included. I'm wondering if I should just remove the <p> (and css defined for it), and put negative margin on the dropdown in the footer instead (instead of positive margin on the <p>).